src/redux/admin/admin.types.js

```diff
@@ -0,0 +1,6 @@
+export const SET_AUTH = 'SET_AUTH';
+export const SET_ADMIN_LOADING = 'SET_ADMIN_LOADING';
+export const SET_ADMIN_ERROR = 'SET_ADMIN_ERROR';
+export const LOGIN_ADMIN = 'LOGIN_ADMIN';
+export const LOGOUT_ADMIN = 'LOGOUT_ADMIN';
+export const CHECK_ADMIN_BY_TOKEN = 'CHECK_ADMIN_BY_TOKEN';
```

Comment:
Please, change `ADMIN` to `USER` because there is only one type of user in this app - admins and we don't need to do any distinction between them. For the application, all admins are just its users